### PR TITLE
Add timezone field and kill dropdowns if there are less than 5 options

### PR DESCRIPTION
**Pull request by Calvinp**
Adds the timezone field to the New Request Form, and turns dropdowns into buttons if there are less than 5 options.
This looks like this:
screenshot 2016-07-29 14 39 09

cc @kwm4385 @tpetr @wolfd

### CODE REVIEW

**Comment by kwm4385** [comment]
Is there an enumeration somewhere of valid timezones we can autocomplete/have the user choose from instead of guessing?

File: SingularityUI/app/components/common/formItems/formGroups/SelectFormGroup.jsx [hunk 1]
```diff
@@ -1,20 +1,46 @@
...
-      className={props.id}
-      options={props.options}
-      onChange={props.onChange}
-      value={props.value}
-      clearable={false}
-    />
-  </FormGroup>
-);
+const SelectFormGroup = (props) => {
+  let selectors;
```
Comment (zdhickman) [inline SingularityUI/app/components/common/formItems/formGroups/SelectFormGroup.jsx]: A more accurate name would relate to the options you're rendering

**Comment by Calvinp** [comment]
Yes, there are lots of enumerations out there. I'm not sure if any of them correspond exactly to the one in Java.utils.TimeZone though.

**Comment by kwm4385** [comment]
Looks like it's listable though `TimeZone.getAvailableIDs();`
While it may be overkill to write an endpoint to get that, perhaps we could run it once and just capture the output.

**Comment by Calvinp** [comment]
This is now implemented.

**Comment by tpetr** [comment]
LGTM

**Pull request merged**